LoopKitTests/InsulinMathTests.swift

```diff
@@ -508,6 +511,22 @@ class InsulinMathTests: XCTestCase {
             XCTAssertEqual(expected.quantity.doubleValue(for: HKUnit.milligramsPerDeciliter), calculated.quantity.doubleValue(for: HKUnit.milligramsPerDeciliter), accuracy: 1.0, String(describing: expected.startDate))
         }
     }
+    
+    func testGlucoseEffectFromTempBasalExponential() {
+        let input = loadDoseFixture("basal_dose_with_delivered")
+        let output = loadGlucoseEffectFixture("effect_from_basal_output_exponential")
+        let insulinSensitivitySchedule = self.insulinSensitivitySchedule
```

Comment:
This was added so there's a corresponding test for one of the tests that I added to DoseStore. I figured since the fixture was already generated, it wasn't a bad idea to add it. Happy to remove if people agree it isn't needed

Reply:
It's actually a remnant from copy-pasting tests - there are like 10 places in other tests where that's there. Removed :)